TwitchPlaysAssembly/Src/Commands/GlobalCommands.cs

```diff
@@ -230,6 +230,11 @@ public static void ReadModuleInformation([Group(1)] string command, [Group(2)] s
 					goto case 1;
 				}
 
+				if (modules.Count > 5) {
+					IRCConnection.SendMessage($"Sorry, there are more than 5 results. Be more specific");
+					return;
+				}
+
 				IRCConnection.SendMessage($"Sorry, there is more than one module matching your search term. They are: {modules.Select(x => $"“{x.moduleDisplayName}” ({x.moduleID})").Join(", ")}", user, !isWhisper);
```

Comment:
```suggestion
				IRCConnection.SendMessage($"Sorry, there is more than one module matching your search term. They are: {modules.Take(5).Select(x => $"“{x.moduleDisplayName}” ({x.moduleID})").Join(", ")}", user, !isWhisper);
```
I think it might be better to just use `.Take(5)` so that a maximum of 5 results are shown instead of not showing any results.